src/xdocs/config_blocks.xml

```diff
@@ -822,6 +822,37 @@ try {
 &lt;/module&gt;
         </source>
 
+		<p>		
+          Examples of single-line statements without braces:
```

Comment:
Javadoc of the check should be updated, it should be consistent with xml doc

Reply:
@strkkk  I've located the Javadoc folder -checkstyle/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/
Can you please guide me on how to update it so that it becomes consistent with xml doc?

Reply:
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java#L81
Here code examples of violations/ok code start, there all lines with examples you added should be added as well.

You can check if it is ok by running locally unit test - com.puppycrawl.tools.checkstyle.internal.XdocsJavaDocsTest#testAllCheckSectionJavaDocs

Reply:
@strkkk I've added the code, still 4 checks are failing can you please help as I've been trying to solve it from a long time now.

Reply:
Is it because I have 3 different commits?

Reply:
@xkarx It depends on which check you are talking about.
I do recommend you squash all your commits into 1 and run the verify command locally and the failures will provide guidance on what needs to be fixed. See https://github.com/checkstyle/checkstyle/wiki/PR-rules

Copy/paste any errors that you are unsure of and we can provide further guidance then.